syntax = "proto3";

package gnss_radar_math;

service GnssMathService {
  rpc ListCoordinates(ListCoordinatesRequest) returns (ListCoordinatesResponse);
}

message Coordinate {
  float x = 1;
  float y = 2;
  float z = 3;
}

message ListCoordinatesRequest {
  string satellite_name = 1;
}

message ListCoordinatesResponse {
  repeated Coordinate coordinates = 10;
}
